src/Xamarin.Android.Build.Tasks/Utilities/FileResourceParser.cs

```diff
@@ -44,7 +46,6 @@ protected XDocument LoadPublicXml () {
 		public IList<R> Parse (string resourceDirectory, IEnumerable<string> additionalResourceDirectories, Dictionary<string, string> resourceMap)
 		{
 			Log.LogDebugMessage ($"Parsing Directory {resourceDirectory}");
-			string publicXmlPath = Path.Combine (JavaPlatformDirectory, "data", "res", "values");
```

Comment:
This line just looked like it was duplicate/extra of the `LoadPublicXml()` method above.